contracts/deployVRC/deployVRC.go

```diff
@@ -127,6 +125,14 @@ func main() {
 			time.Sleep(1 * time.Second)
 		}
 
+		receipt, err := client.TransactionReceipt(context.Background(), tx.Hash())
```

Comment:
Everything looks good, one final thing can you just replace `context.Background()` with the `cli.Context`

Reply:
@nisdas I dont think that works `cli.Context` doesn't satisfy `context.Context`, it's missing `Deadline` method

Tried:
```
ctx := cli.NewContext(app, nil, nil)
receipt, err := client.TransactionReceipt(ctx, tx.Hash())
```
Got:
```
*cli.Context does not implement context.Context (missing Deadline method)
```

Reply:
Ah , k nvm looks like the context types are different. The just do `ctx := context.Background()` and use that whenever a context is required in the main function

Reply:
Yup. Can you approve if there's nothing else?